packages/example-contracts/contracts/cross-chain-warriors/CrossChainWarriors.sol

```diff
@@ -33,12 +33,16 @@ contract CrossChainWarriors is
 
     Counters.Counter public tokenIds;
 
+    ZetaTokenConsumer private _zetaConsumer;
+
     constructor(
         address connectorAddress_,
         address zetaTokenAddress,
+        address zetaConsumerAddress,
         bool useEven
     ) ZetaInteractor(connectorAddress_) {
         _zetaToken = IERC20(zetaTokenAddress);
+        _zetaConsumer = ZetaTokenConsumer(zetaConsumerAddress);
```

Comment:
Nit:

```suggestion
        address zetaConsumer_,
        bool useEven
    ) ZetaInteractor(connectorAddress_) {
        _zetaToken = IERC20(zetaTokenAddress);
        _zetaConsumer = ZetaTokenConsumer(zetaConsumer_);
```

Reply:
connectorAddress_ needs to be updated. done